src/cmdlib.sh

```diff
@@ -462,12 +469,20 @@ EOF
     else
         rm -vf "${local_overrides_lockfile}"
     fi
+
+    contentset_path=""
     if [ -e "${configdir}/content_sets.yaml" ]; then
+        contentset_path="${configdir}/content_sets.yaml"
+    elif [ -e "${workdir}/src/yumrepos/content_sets.yaml" ]; then
```

Comment:
This will have to be adapted for variants too

Reply:
We've discussed that and we've realized that we can just merged all content_sets file into a single one without issue, thus avoiding the need for variants.